model-optimizer/mo/ops/slice.py

```diff
@@ -13,133 +13,181 @@
+operations to OpenVINO operations from opset:
+ - AttributedSlice, TFSlice -> Slice 
+ - CaffeSlice -> Split 
+ - MXSlice -> StridedSlice 
+"""
+
+
+class AttributedSlice(Op):
+    """
+    AttributedSlice is used in old versions of ONNX models (opset version < 10).
+    The operation is replaced with the OpenVINO Slice operation on the front phase.
+    """
+    op = 'AttributedSlice'
+    enabled = False
+
+    def __init__(self, graph: Graph, attrs: dict):
+        super().__init__(graph, {
+            'type': None,
+            'op': self.op,
+            'in_ports_count': 1,
+            'out_ports_count': 1,
+            'infer': None,
+        }, attrs)
+
+    def supported_attrs(self):
+        return ['axes', 'starts', 'ends']
+
+
+class CaffeSlice(Op):
+    """
+    Slice in Caffe is equivalent to Split operation in OpenVINO.
+    https://caffe.berkeleyvision.org/tutorial/layers/slice.html
+    The operation is replaced with the OpenVINO Split operation on the front phase.
+    """
+    op = 'CaffeSlice'
+    enabled = False
+
+    def __init__(self, graph: Graph, attrs: dict):
+        super().__init__(graph, {
+            'type': None,
+            'op': self.op,
+            'in_ports_count': 1,
+            'out_ports_count': 1,
+            'infer': None,
+        }, attrs)
+
+    def supported_attrs(self):
+        return ['slice_point', 'axis']
+
+
+class TFSlice(Op):
+    """
+    Slice operation in Tensorflow is different from Slice in ONNX, Caffe and MXNet. It has begin and size inputs while
+    OpenVINO Slice has start, end, step and axis parameters specified as inputs.
+    https://www.tensorflow.org/api_docs/python/tf/slice
+    The operation is replaced with the OpenVINO Slice operation on the front phase.
+    If size[i] == -1 is replaced to int32_max value for the end.
+    """
+    op = 'TFSlice'
+    enabled = False
+
+    def __init__(self, graph: Graph, attrs: dict):
+        super().__init__(graph, {
+            'type': None,
+            'op': self.op,
+            'in_ports_count': 3,
+            'out_ports_count': 1,
+            'infer': None,
+        }, attrs)
+
+
+class MXSlice(Op):
+    """
+    Slice operation in MXNet is different from ONNX, Caffe, Tensorflow. It has begin, end & step attributes
+    https://mxnet.apache.org/versions/1.6/api/python/docs/api/symbol/op/index.html#mxnet.symbol.op.slice
+    The operation is replaced with the OpenVINO StridedSlice operation on the front phase.
+    """
+    op = 'MXSlice'
+    enabled = False
+
+    def __init__(self, graph: Graph, attrs: dict):
+        super().__init__(graph, {
+            'kind': 'op',
+            'type': None,
+            'op': self.op,
+            'in_ports_count': 1,
+            'out_ports_count': 1,
+            'infer': None
+        }, attrs)
 
 
 class Slice(Op):
+    """
+    Semantic of OpenVINO Slice operation is identical ONNX Slice (opset >= 10).
```

Comment:
There is no OpenVINO Slice operation. It is not in public opset

Reply:
Updated comments. Please review.